orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -61,7 +66,7 @@ class DependentPipelineStarter implements ApplicationContextAware {
 
     log.info('triggering dependent pipeline {}:{}', pipelineConfig.id, json)
 
-    User principal = getUser(parentPipeline)
+    User principal = propagateAuthentication ? getUser(parentPipeline) : null
```

Comment:
What about the case where both pipelines have the same permissions? This will break those users